### PR TITLE
Add Prometheus to module dependencies

### PR DESCRIPTION
**Pull request by linux019**
I need to scrape metrics from the modules, so Prometheus has been added to the module dependencies. PBS doesn't use default Prometheus Gatherer, so the only way is to pass the same instance of Prometheus.Gatherer to modules. Please review

### CODE REVIEW

**Review by hhhjort — APPROVED** [review]
Looks reasonable to me.

**Review by VeronikaSolovei9 — DISMISSED** [review]

**Comment by bsardo** [comment]
Hi @linux019, it appears you have interest in logging metrics from modules. What kind of metrics are you interested in logging? Are they global metrics or bidder/account level metrics?

Issue [Allow modules to log metrics](https://github.com/prebid/prebid-server/issues/2914) was opened a while back and there is a discussion there about the requirements. Please give it a read and chime in with your thoughts.
Basically, we are ok with global metrics from modules at this time and others have expressed an interest in bidder/account level metrics but the design for that still needs to be fleshed out and signed off by the Prebid server committee.

Also any attempt to use a metrics collector in modules should be an interface instead of tying it to a specific technology.

**Comment by bretg** [comment]
This enhancement is tracked by https://github.com/prebid/prebid-server/issues/2914 -- the committee has not agreed to provide direct metrics access to modules, especially one particular metrics system. 

Sorry @linux019 , but for now you should make this change in your fork and help us hash out a better long term solution in the issue.
